### ISSUE TITLE
ConsoleLoggerConfigurationExtensions in both Serilog.Sinks.Console and Serilog.Sinks.Debug

### ISSUE THREAD

**Issue by jonathantsao** [serilog/serilog-aspnetcore#227]
My project has Serilog.AspNetCore as a dependency, and I wish to use the `ConsoleLoggerConfigurationExtensions` extension methods from the `Serilog.Sinks.Console` package that is referenced within `Serilog.AspNetCore` package. However, I get this build error message:

```
The type 'ConsoleLoggerConfigurationExtensions' exists in both
'Serilog.Sinks.Console, Version=3.1.1.0, Culture=neutral
and 'Serilog.Sinks.Debug, Version=1.0.1.0, Culture=neutral
```

Looking at both GitHub repos [Serilog.Sinks.Debug](https://github.com/serilog/serilog-sinks-debug/blob/dev/src/Serilog.Sinks.Debug/DebugLoggerConfigurationExtensions.cs) and [Serilog.Sinks.Console](https://github.com/serilog/serilog-sinks-console/blob/dev/src/Serilog.Sinks.Console/ConsoleLoggerConfigurationExtensions.cs):
Both projects declare a class named `ConsoleLoggerConfigurationExtensions`, and I'm wondering how `Serilog.AspNetCore` compiled in the first place. 

**Relevant package, tooling and runtime versions**
Using net5.0 with Serilog.AspNetCore 3.4.0

Note: I reported this issue in this repo because the issue wouldn't present itself individually in either package, only in the dependency of both packages.

**Comment by nblumhardt** [comment]
Thanks! Crazy that that one slipped under the radar for so long. Re-publishing _Serilog.Sinks.Debug_ now.

**Comment by nblumhardt** [comment]
Hi @jonathantsao  - package `Serilog.Sinks.Debug` `2.0.0-dev-00026` has now been published to NuGet; if you add an explicit reference to that package from your top-level project, is the problem resolved successfully? Thanks!

**Comment by nblumhardt** [comment]
Fixed now in #234
